jira-config/jira-config-api/pom.xml

```diff
@@ -51,5 +51,41 @@
       <version>${commons.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+      <version>5.2</version>
+    </dependency>
+    <dependency>
```

Comment:
Since several different modules depend on this I would at the very least put a Maven property to store the version in the root pom of the jira project.

Reply:
Not sure what you mean here, I don't remember suggesting to remove any exclusion. As I said, one benefit of setting up a dependency management is that you can put the exclusions in this central place, and it will apply to all modules where you have this dependency.

Reply:
Didn't full understood what you was thinking but now it should be good.